style.css

```diff
@@ -32,7 +31,6 @@ html, body {
     font-family: var(--main-title-font);
```

Comment:
Why are you declaring `font-family` two times?

Reply:
Right, if main font is in the body, I can delete it from others places.
Fixed.